requirements.txt

```diff
@@ -1,7 +1,10 @@
 attrs==19.3.0
+bcrypt==3.1.7
```

Comment:
Are these changes to the requirements necessary? It doesn't seem like you ultimately decided to use bcrypt.